api/v1alpha1/clusterextension_types.go

```diff
@@ -303,9 +295,10 @@ type CatalogSource struct {
 	//
 	// [RFC 1123]: https://tools.ietf.org/html/rfc1123
 	//
-	//+kubebuilder:validation:items:MaxLength:=253
-	//+kubebuilder:validation:items:Pattern:=^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
-	//+optional
+	// +kubebuilder:validation:items:MaxLength:=253
+	// +kubebuilder:validation:MaxItems:=256
```

Comment:
I don't see mention of the max items in the GoDoc for this field. Should we include it to ensure it is communicated to a user doing something like `kubectl explain ...`?